docs/docs/how-to-install-packages.md

```diff
@@ -0,0 +1,21 @@
+---
+title: How to install packages
+slug: /how-to-install-packages
+tags:
+  - lerna
+  - packages
+---
+
+# Best practices for installing packages in Strapi
+
+When working with the Strapi mono repo, it's important to follow best practices for installing packages to avoid potential issues and ensure consistent results. Instead of using the standard **`yarn add`** command, we recommend using **`yarn lerna add <package_name> --scope @strapi/<module_name>`** for installing packages.
+
+This approach uses Lerna, a tool for managing JavaScript projects with multiple packages, to ensure that the package is installed in the correct location and version across all modules. The **`--scope`** flag specifies the specific module that the package should be installed in, ensuring that it's only installed where it's needed.
+
+By using this method, Strapi developers can avoid issues with mismatched package versions or unnecessary dependencies in certain modules. This can help to keep the codebase clean and maintainable, and reduce the potential for conflicts or issues in the future.
+
+Overall, we recommend using **`yarn lerna add`** with the **`--scope`** flag for installing packages in the Strapi mono repo, to ensure consistent and reliable results.
```

Comment:
Can we include a message with the error that may be received without lerna, so that searches for the error will find it?

Reply:
I guess we can, if I can find it in Slack 😂